packages/docs/celo-codebase/protocol/identity/privacy-research.md

```diff
@@ -0,0 +1,14 @@
+# Future Privacy Research
```

Comment:
Mostly just the same content as privacy.md but renamed to privacy-research since it covers future work